integration-webcert/src/test/java/se/inera/intyg/minaintyg/integration/webcert/converter/metadata/MetadataConverterTest.java

```diff
@@ -160,9 +167,112 @@ void shallReturnNullIfNoRecipient() {
             .unitName(UNIT_NAME)
             .build())
         .created(ISSUED)
+        .summary(CertificateSummary.builder()
+            .label(SUMMARY_LABEL)
+            .value(SUMMARY_VALUE)
+            .build())
         .build();
 
     final var actualMetadata = metadataConverter.convert(metadataDTONoRecipient);
     assertNull(actualMetadata.getRecipient());
   }
+
+  @Test
+  void shallConvertSummaryLabel() {
+    final var actualMetadata = metadataConverter.convert(metadataDTO);
+    assertEquals(SUMMARY_LABEL, actualMetadata.getSummary().getLabel());
+  }
+
+  @Test
+  void shallConvertSummaryValue() {
+    final var actualMetadata = metadataConverter.convert(metadataDTO);
+    assertEquals(SUMMARY_VALUE, actualMetadata.getSummary().getValue());
+  }
+
+  @Test
+  void shallReturnNullIfNoSummaryLabel() {
```

Comment:
This comment is for this test and the tests below.
1) See if you can simplify the test methods to not have to create the whole metadataDTO just to test things related to summary e.g. No summary label.
2) Consider changing the the asserts for these tests... instead of verifying that the label is null you verify that the summary is empty or partially filled with data.